middle_end/flambda2/terms/code_metadata.ml

```diff
@@ -125,6 +128,7 @@ type 'a create_type =
   contains_no_escaping_local_allocs:bool ->
   stub:bool ->
   inline:Inline_attribute.t ->
+  check:Lambda.check_attribute ->
```

Comment:
This should have a new module `Check_attribute`, like `Inline_attribute`, to decouple from Lambda please.  Then `Check_attribute.t` here.